src/components/actionSheet/index.js

```diff
@@ -59,6 +59,27 @@ export default class ActionSheet extends BaseComponent {
      * When passed (only with useNativeIOS), will display a cancel button at the bottom (overrides cancelButtonIndex)
      */
     showCancelButton: PropTypes.bool,
+    /**
+     * Add or override style of the action sheet
+     */
+    sheetStyle: ViewPropTypes.style,
```

Comment:
Is sheelStyle is a containerStyle?

Reply:
It's a container of the title and actions, but it is a view inside the dialog, do you think containerStyle is a better name?

Reply:
I do. We use 'containerStyle' when the style implied on the wrapping views of the component

Reply:
Done